src/build-extensions-oscontainer.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#used by cmd/build-extensions-container.go
+RHCOS_IMAGE=registry.ci.openshift.org/rhcos-devel/rhel-coreos:latest
+#Find the RHCOS ociarchive.
+RHCOS_OCIARCHIVE=$(find ~+ -name "rhcos*ociarchive")
+cd src/config
+#Replace the FROM line with the ociarchive
+#FROM oci-archive:/srv/builds/VERSION/x86_64/rhcos-VERSION.ociarchive as os
+sed -i "s|$RHCOS_IMAGE|oci-archive:$RHCOS_OCIARCHIVE|" extensions/Dockerfile
```

Comment:
I think we can just use `podman build --from oci-archive:$RHCOS_OCIARCHIVE`

(Also tangentially in theory this isn't specific to RHCOS, so in the variable names and such we can just say e.g. `ostree_ociarchive` or something)